adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DeviceRegistryBasedPskStore.java

```diff
@@ -194,14 +194,15 @@ public PskSecretResult requestPskSecretResult(
             final PskPublicInformation identity,
             final String hmacAlgorithm,
             final SecretKey otherSecret,
-            final byte[] seed) {
+            final byte[] seed,
+            final boolean useExtendedMasterSecret) {
```

Comment:
useExtendedMasterSecret does not seem to be used?

Reply:
Yes, it's not used. But it's part of the [callback API](https://github.com/eclipse/californium/blob/master/scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/pskstore/AdvancedPskStore.java#L61-L62).
Any rules you want to be obeyed for not used parameter in callbacks?

Reply:
Ah, my bad. I didn't see that this has an `@Override` annotation ...